weaviate/batch/reference.go

```diff
@@ -39,3 +50,11 @@ func (rb *ReferencesBatcher) Do(ctx context.Context) ([]models.BatchReferenceRes
 	decodeErr := responseData.DecodeBodyIntoTarget(&batchResponse)
 	return batchResponse, decodeErr
 }
+
+func (rb *ReferencesBatcher) buildPath() string {
+	path := "/batch/references"
+	if rb.consistencyLevel != "" {
+		path = fmt.Sprintf("%s?consistency_level=%v", path, rb.consistencyLevel)
+	}
+	return path
+}
```

Comment:
total nitpick:
I've seen this piece of code 3 times, could we somehow combine it?

Reply:
Refactored into its own package which handles path building across the codebase